packages/mds-ingest-service/repository/index.ts

```diff
@@ -199,14 +219,61 @@ class IngestReadWriteRepository extends ReadWriteRepository {
         query.andWhere('events.provider_id = ANY(:provider_ids)', { provider_ids })
       }
 
-      const entities = await query.limit(limit).getMany()
+      // Use query instead of paginator to manage order if using a joined field
```

Comment:
couldn't figure out how to pass an order column string for a joined field to buildPaginator without it throwing an error, if there's a better way to handle this field happy to change it.